tempesta_fw/http.c

```diff
@@ -1762,6 +1789,10 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 	if (r < 0)
 		return r;
 
+	r = tfw_http_add_loc_hdrs(hm, req);
+	if (r < 0)
+		return r;
```

Comment:
Not only requests are must be adjustable - an example from the issue `hdr_add Expires "Thu, 15 Apr 2020 20:00:00 GMT"` is about responses.

Reply:
Actually only responses are adjusted in the patch: `tfw_http_add_loc_hdrs()` function is called in the `tfw_http_adjust_resp()` as a part of response processing. Is it required to adjust _requests_ before forwarding them to backend servers?

Reply:
Yeah, my bad - of course I was commenting `tfw_http_adjust_resp()` :)

We absolutely have to be able to adjust requests before forwarding them to a backend. `X-Forwarded-For` is the most trivial example which we hard code for now. Other examples could be security tokens passed to most powerful WAFs or removing some danger headers from suspicious requests.